themes/inbeat/layouts/partials/site-header.html

```diff
@@ -126,16 +126,20 @@ <h2>{{ .topText }}</h2>
 						<a href="https://inbeat.agency/" target="_blank" title="Inbeat Agency">The Agency</a>
 					</li>
 					<li class="creator-studio">
-						<a href="{{- $.Site.BaseURL -}}ugc-studio" title="UGC Studio">UGC Studio</a>
+						<a href="{{- $.Site.BaseURL -}}ugc-platform" title="UGC Platform">UGC Platform</a>
 						<span class="new-badge">NEW</span>
 					</li>
 				</ul>
 			</div>
 			<div class="col-xs-4 col-md-3 hide-sm">
 				<ul class="right">
 					<li class="cta">
-						<a class="btn btn-lg no-background" href="https://app.inbeat.co"><span>Sign in</span></a>
-						<a href="https://app.inbeat.co/get-started" title="Sign up" class="btn btn-sm btn-crazy-hover no-border" target="_blank">Sign up</a>
+            {{ if eq .RelPermalink "/ugc-platform/" }}
```

Comment:
Is the trailing slash always there? I don't remember how I usually checked the page we were on but this way is nice.